spec/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in tictactoe.gemspec
+gemspec
```

Comment:
Let's keep this Gemfile at the root directory, not `spec/` directory. 

Also, is this line valid? What is it for? :thinking: